docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -166,8 +166,9 @@
      "name": "stderr",
```

Comment:
<p>Check your machine configuration and re-run to get rid of <code>ERROR 1: PROJ: proj_create_from_database: Open of /opt/conda/share/proj failed</code>. This is an internal error probably in geopandas. PROJ issues this error when it can't find it's sqlite database. This is specified in a config or env variable somewhere in geopandas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
removed